string name
uint32 pid
string hostname
float32 cpu
uint64 memory
uint32 error
uint32 NO_ERROR=0
uint32 ERROR_PID_NOT_FOUND=1
